gateway/cors.go

```diff
@@ -24,6 +24,10 @@ var (
 
 	corsExposedHeaders = []string{
 		fhttp.HeaderXCorrelationID,
+		fhttp.HeaderXTotal,
+		fhttp.HeaderXTotalPages,
+		fhttp.HeaderXPerPage,
+		fhttp.HeaderXPage,
```

Comment:
These should be sorted alphabetically, as well as the other through this PR.

Reply:
Fixed